pkg/config/default-config.yml

```diff
@@ -76,6 +76,9 @@ defaultEnvironment:
 
   MANAGE_HOST: manage
   MANAGE_PORT: 9008
+  MANAGE_AUTH_PASSWORD_FILE: /run/secrets/manage_auth_password
+
+  OPENSLIDES_DEVELOPMENT: 0
```

Comment:
I don't know what the default value should be. I think you default value is not the number 0 but the string "0", which could be confusing.

I choose the string "false": https://github.com/OpenSlides/openslides-autoupdate-service/blob/69d94f1595f15243c1d78d06037cf6eba25f13b6/cmd/autoupdate/main.go#L60

Reply:
Done.